### DISCUSSION TITLE
速度打五折了，勉强能播放！大家呢？

### DISCUSSION THREAD

**Discussion by kaixinxiaowangzi666** [AlistGo/alist#5636] [General]
个人怀疑不是官方接口限速，那么小白羊又是正常的，大家是只能恢复的？

**Comment by wzls16** [comment]
可能是专门针对webdav限速的

**Reply by kaixinxiaowangzi666** [reply]
不是，下载速度打折扣了

**Reply by zhoutao-cn** [reply]
![image](https://github.com/alist-org/alist/assets/126696101/118fa7be-c521-489e-834f-cd8f6434a888)

**Reply by zhoutao-cn** [reply]
小白羊满速，Alist 被限速到4M。

**Reply by kaixinxiaowangzi666** [reply]
还不知多久能修复到以前
